File: src/main/java/com/bot/commands/NSFWCommand.java
package com.bot.commands;

import com.bot.utils.CommandCategories;
import net.dv8tion.jda.api.Permission;

public abstract class NSFWCommand extends BaseCommand {

    public NSFWCommand() {
        this.category = CommandCategories.NSFW;
        this.guildOnly = false;
        this.botPermissions = new Permission[]{Permission.MESSAGE_WRITE, Permission.MESSAGE_EMBED_LINKS};
    }
}

File: src/main/java/com/bot/commands/NSFWCommand.kt
package com.bot.commands

import com.bot.utils.CommandCategories
import net.dv8tion.jda.api.Permission

abstract class NSFWCommand : BaseCommand() {
    init {
        this.category = CommandCategories.NSFW
        this.guildOnly = false
        this.botPermissions = arrayOf(Permission.MESSAGE_WRITE, Permission.MESSAGE_EMBED_LINKS)
    }
}
